sdk/platform/posix/CMakeLists.txt

```diff
@@ -1,8 +1,6 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # SPDX-License-Identifier: MIT
 
-if (NOT WIN32)
```

Comment:
can we change just this single line to "if (UNIX)"? It should work.

Reply:
I am retracting this proposal, it would work, but I think checking AZ_PLATFORM_IMPL gives us more flexibility.